clang/include/clang/Sema/Sema.h

```diff
@@ -67,6 +67,7 @@
 #include "llvm/ADT/SmallPtrSet.h"
 #include "llvm/ADT/SmallSet.h"
 #include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringExtras.h"
```

Comment:
What do we need this addition for?

Reply:
For `toString(*I, 10, false)` in `CheckUInt32Argument` definition that I moved.

Reply:
Got it, thanks!